### ISSUE TITLE
🐞 SvelteKit Recipe needs an update

### ISSUE THREAD

**Issue by uvulpos** [gofiber/recipes#1399]
The SvelteKit Example (https://github.com/gofiber/recipes/tree/master/sveltekit-embed/frontend/src/routes) won't work with newer versions anymore

**Comment by gaby** [comment]
@uvulpos I try updated it but I have limited knowledge in `Sveltekit`. I'm doing a PR that will remove this recipe unless someone wants to update it and submit a PR.

I was able to install all the dependencies, but the `build`  command doesn't work at all, and this code is from 2 years ago

**Comment by uvulpos** [comment]
@gaby thanks for thanking over, I have currently not the time to do it, but its easy.

Just use the SvelteKit Static Adapter and configure it by placing these variables in the layout.svelte
```ts
// import adapter from '@sveltejs/adapter-auto';
import adapter from '@sveltejs/adapter-static';
import { vitePreprocess } from '@sveltejs/kit/vite';

/** @type {import('@sveltejs/kit').Config} */
const config = {
	// Consult https://kit.svelte.dev/docs/integrations#preprocessors
	// for more information about preprocessors
	preprocess: vitePreprocess(),

	kit: {
		// adapter-auto only supports some environments, see https://kit.svelte.dev/docs/adapter-auto for a list.
		// If your environment is not supported or you settled on a specific environment, switch out the adapter.
		// See https://kit.svelte.dev/docs/adapters for more information about adapters.
		adapter: adapter({
            // default options are shown. On some platforms
            // these options are set automatically — see below
            pages: '../golang/src/assets/frontend',
            assets: '../golang/src/assets/frontend',
            fallback: "index.html",
            precompress: false,
            strict: false,
        })
	}
};

export default config;
```
```ts
<script>
  export const prerender = false;
  export const ssr = false;
  export const trailingSlash = "always";
</script>
```

then you can import it in go as a embed
```
//go:embed frontend/*
var SvelteFS embed.FS

func something() {
        publicFS, err := fs.Sub(SvelteFS, "frontend")
	if err != nil {
		log.Fatal(err)
	}

        router.Use("/", filesystem.New(filesystem.Config{
			Root:         http.FS(publicFS),
			NotFoundFile: "index.html",
	}))
}
```

**Comment by uvulpos** [comment]
This is from a project, where I already implemented it

**Comment by yogenpoonudurai** [comment]
Is this issue still active?

**Comment by gaby** [comment]
@yogenpoonudurai I believe it was fixed via #1731

**Comment by yogenpoonudurai** [comment]
Cool. I was looking for an issue to contribute. Glad this was fixed. Thanks @gaby
